### PR TITLE
done to add new section of about the project 

### PR DESCRIPTION
**Pull request by Abubakar-Meigag**
also I edit some styling in the others component

### CODE REVIEW

File: client/src/components/about1/AboutPage.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <div className="hero-content text-center text-neutral-content ">
+          <div className="max-w-lx">
+            <h1 className="text-[45px] font-medium  text-sky-50">
+              Welcome to <span className="underline font-bold">DEJA</span> Review 
+            </h1>
+            <h1 className="mb-6 text-[32px] font-medium underline  text-sky-50">
+              Spaced Repetition technique
+            </h1>
+
+            <div className="text-2xl mb-6 text-sky-50 font-semibold">
```
Comment (KristinaDudnyk) [inline client/src/components/about1/AboutPage.jsx:25]: It was a cool idea to add the motto to the top of this page. But it looks too static. 
Could you consider implementing a feature that would animate this text like:
**Where you always**(text stay static) and other part is animated to change every 2ish second

File: client/src/components/about1/Content.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        <div className=""></div>
+        <div className="hero-content text-center text-neutral-content">
+          <div className="max-w-lx">
+            <h1 className="mb-2 text-2xl text-left font-bold">
+              What is Spaced Repetition?
+            </h1>
+
+            <div>
+              <div className="w-[60rem] text-xl text-justify">
+                <div className="">
```
Comment (KristinaDudnyk) [inline client/src/components/about1/Content.jsx:23]: The About section should focus on **what this app is bringing to the user** , **what solution it offers** 
Instead, this is more like a Wikipedia article about what Spaced repetition is

File: client/src/components/about1/aboutPage.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Anton&family=Inclusive+Sans:ital@0;1&family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Monomaniac+One&family=Mooli&family=Quicksand:wght@300&family=Roboto+Slab:wght@300&family=Roboto:ital,wght@0,300;1,400&display=swap");
+
+h1 {
+  font-family: "Merriweather", serif;
+}
```
Comment (KristinaDudnyk) [inline client/src/components/about1/aboutPage.css:5]: For some reason, the changes are overwriting the css of the sidebar
The sidebar's font code should be the same

image

File: client/src/components/about1/Demo.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                  </span>
+                  <h2>Part 2</h2>
+                  <span>
+                    How to create new topic from CYF modules?. How to add it
+                    into your dashboard?. & How to review the topic?{" "}
+                  </span>
+                </p>
+              </div>
+            </div>
+          </div>
```
Comment (KristinaDudnyk) [inline client/src/components/about1/Demo.jsx:32]: This section doesn't bring much to the user and it doesn't read easily 
Please, consider removing it completely or swapping it for something more valuable

File: client/src/components/about1/AboutPage.jsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+function AboutPage() {
+  return (
+    <div>
+      <div 
+        className="hero min-h-screen"
+        style={{
+          backgroundImage:
+            "url(https://daisyui.com/images/stock/photo-1507358522600-9f71e620c44e.jpg)",
+        }}
```
Comment (KristinaDudnyk) [inline client/src/components/about1/AboutPage.jsx:13]: This background image looks out of context. It doesn't go well with our design 
Consider swapping it for something like the icons we used for the sidebar. Simple lines, simple colours, something that will complement the aesthetic we have.

**Review by KristinaDudnyk — CHANGES_REQUESTED** [review]
With the next pull request, please provide a screnshots of the changes you made

**Comment by Abubakar-Meigag** [comment]
done all the comment and styling 

ready to marge

**Pull request merged**